pinot-spi/src/main/java/org/apache/pinot/spi/config/table/TableConfig.java

```diff
@@ -288,8 +303,8 @@ public void setTierConfigsList(List<TierConfig> tierConfigsList) {
   }
 
   @JsonIgnore
-  public UpsertConfig.HashFunction getHashFunction() {
-    return _upsertConfig == null ? UpsertConfig.HashFunction.NONE : _upsertConfig.getHashFunction();
+  public HashFunction getHashFunction() {
```

Comment:
We should remove this method. The hash function can come from both upsert config and dedup config

Reply:
Ack